arangod/Agency/RestAgencyHandler.h

```diff
@@ -25,10 +25,38 @@
 #define ARANGOD_REST_HANDLER_REST_AGENCY_HANDLER_H 1
 
 #include "Agency/Agent.h"
+#include "Futures/Future.h"
+#include "Futures/Unit.h"
 #include "RestHandler/RestVocbaseBaseHandler.h"
 
 namespace arangodb {
 
+// String to number conversions:
```

Comment:
Do we not have all these in `lib/Basics/StringUtils.*`?

Reply:
not generalized so that one call like stp below gives you all. i am happy to carry these over to stringutils.

Reply:
that's what i am doing?

Reply:
nice i got it.